idf_build_apps/args.py

```diff
@@ -191,6 +192,15 @@ class DependencyDrivenBuildArguments(GlobalArguments):
         validation_alias=AliasChoices('manifest_files', 'manifest_file'),
         default=None,
     )
+    manifest_filepatterns: t.Optional[t.List[str]] = field(
+        FieldMetadata(
+            validate_method=[ValidateMethod.TO_LIST],
+            nargs='+',
+        ),
+        description='space-separated list of file patterns to search for the manifest files. '
```

Comment:
Is there a reason to use space-separated lists for this argument when modified_components and modified_files use semicolon-separated lists?

Reply:
using semicolon-separated lists in order to differentiate these scenarios

- not specified at all -> treat as `None`
- specified empty string -> treat as `None`
- specified ";" -> treat as `[]`
- specified some values

Here using space-separated list since we don't have to differentiate `None` with `[]`, and space feels more natural together with nargs="+"